src/Localization.jsx

```diff
@@ -24,15 +25,9 @@ const Localization = React.createClass({
     };
   },
 
-  localize(key, ...values) {
-    const { messages } = this.props;
-    let string = get(messages, key, key);
-
-    if (values && string) {
-      string = util.format(string, ...values);
-    }
-
-    return string;
+  localize(key, values, defaultValue) {
+    const { messages, localize } = this.props;
+    return localize(messages, key, values, defaultValue);
```

Comment:
The defaultValue most of the times is optional, but key and values are more likely to be present. To be honest I was following lodash.get and java message bundles pattern here.